src/consts/branding.ts

```diff
@@ -0,0 +1,6 @@
+import { WalletName } from '@solana/wallet-adapter-base';
+
+export const CROSSMINT_LOGO_21x21 =
```

Comment:
Just curious. Are these images in base64 for any particular reason? Can't they be svgs?

Reply:
Not sure if we have an svg version of our logo. If we do, and its smaller than the png ill def use that